util.py

```diff
@@ -2,19 +2,11 @@
 import geopy
 import pyipinfodb
 import requests
-import sys
-sys.path.insert(0, 'cache')
 
 from geopy.distance import great_circle
 
-import ip_location_cache
-
-def get_distance(ip_addr1, ip_addr2):
-    ip_cache = ip_location_cache.ip_location_cache()
-    location1 = ip_cache.get_lat_lon_from_ip(ip_addr1)
-    location2 = ip_cache.get_lat_lon_from_ip(ip_addr2)
-    print location1
-    print location2
+# get distance from two (lat,log) pairs
+def get_distance(location1, location2):
```

Comment:
@codingjaguar I think this is used in a lot of different files for IP, so if you want to just find the lat/lon distance, it might be better to make a new function